docs/HowTo/Get-Started/Snapshot-Start.md

```diff
@@ -0,0 +1,34 @@
+---
+description: How to start Teku using a recent finalized checkpoint state
+---
+
+# Start Teku from a recent state
+
+You can start Teku from a recent finalized checkpoint state to avoid syncing a node from a
+historic state.
```

Comment:
This phrase "to avoid syncing a node from a historic state" doesn't quite make sense to me.  What about:
```suggestion
To get Teku up and running in only a few minutes, you can start Teku from a recent finalized checkpoint state rather than syncing from genesis.
```

Reply:
Yes, your suggestion sounds much better.